.maestro/subflows/repay.yaml

```
@@ -1,19 +1,32 @@
+- evalScript: ${output.debt = Number(maestro.copiedText)}
 - copyTextFrom:
     text: "[^%.]+"
     below: "${maestro.copiedText}"
     leftOf: Repay
 - evalScript: ${output.maturity = maestro.copiedText}
 - tapOn: { text: Repay, rightOf: "${output.maturity}" }
+- waitForAnimationToEnd
 - tapOn: { text: Repay, leftOf: Rollover }
-- tapOn: Max
+- runFlow:
+    when: { true: "${!JSON.parse(partial ?? false)}" }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)